include/eve/traits/value_type.hpp

```diff
@@ -67,5 +67,6 @@ namespace eve
   //================================================================================================
 
   template <typename T>
+    requires (!value<T>)
```

Comment:
why this change ?

Reply:
It tries to get the value of doible and that results in a hard error, not sfinae error.

I don't have a concept that'd distinguish value and pointer here. So I did !value

Reply:
OK